src/commands/invite.ts

```diff
@@ -25,6 +27,15 @@ export async function runInviteCommand(
   const groups: string[] = userGroups?.length ? userGroups : defaultGroups
   const username: string = await getUserDisplayName(client, userId)
   const rooms: RoomsList = getRoomsByGroups(groups)
+
+  if (!userId.includes(`:${IDENTITY_SERVER}`)) {
+    // userId is something like "@username:identity.server.org"
+    const [, wrongIdentityServer] = userId.split(":")
+    throw new CommandError(
+      `This handle is not registered under ${IDENTITY_SERVER}, but ${wrongIdentityServer}. \nMake sure that username ends with ":${IDENTITY_SERVER}"`,
+    )
+  }
```

Comment:
I feel like it should be "homeserver", instead of "identity server". Identity servers are lookup services, which clearly what you didn't mean here.

Reply:
[0de621d](https://github.com/paritytech/matrix-admin-bot/pull/42/commits/0de621d71ba3b156d7a120eac54e372a7d24ec03)